applications/GeoMechanicsApplication/custom_processes/geo_integration_values_extrapolation_to_nodes_process.h

```diff
@@ -0,0 +1,159 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ \.
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main authors:    Vicente Mataix Ferrandiz
+//                   Jonathan Nuttall
+//                   Wijtze Pieter Kikstra
+//                   Richard Faasse
+
+#pragma once
+
+#include "geometries/geometry.h"
+#include "includes/element.h"
+#include "includes/key_hash.h"
+#include "includes/kratos_parameters.h"
+#include "includes/node.h"
+#include "processes/process.h"
+
+#include <unordered_map>
+
+namespace Kratos
+{
+
+class Model;
+class ModelPart;
+class NodalExtrapolator;
+
+class ProcessInfo;
+
+/**
+ * @class IntegrationValuesExtrapolationToNodesProcess
+ * @ingroup GeoMechanicsApplication
+ * @brief This process extrapolates vales from the integration points to the nodes
+ * @details This process solves local problems in order to extrapolate the values from the gauss point to the nodes. Uses inverse for same number of nodes and GP and generalized inverse for cases where the number of GP in higher than the number of nodes
+ * Using as main reference: https://www.colorado.edu/engineering/CAS/courses.d/IFEM.d/IFEM.Ch28.d/IFEM.Ch28.pdf (Felippa Stress Recovery course)
+ * @author Vicente Mataix Ferrandiz
+ */
+class KRATOS_API(GEO_MECHANICS_APPLICATION) GeoIntegrationValuesExtrapolationToNodesProcess : public Process
+{
+public:
+    using NodeType     = Node;
+    using GeometryType = Geometry<NodeType>;
```

Comment:
I would remove the `NodeType` alias and replace its single usage by just `Node`:
```suggestion
    using GeometryType = Geometry<Node>;
```

Reply:
Done